learn/react-native/integrating-crashlystics.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Integrating Crashlystics"
+id: "integrating-crashlystics"
+sidebar_label: "Crashlystics"
+---
+---
+import crashLogs from '/learn/assets/CrashLogsAttributesAndErrorReports.png';
+
+Crash reporting services like Crashlytics are essential tools for modern app development. They provide real-time error monitoring, insightful crash reports, and streamlined debugging processes. By monitoring for issues and proactively addressing them, developers can create a more stable and reliable app environment, leading to higher user satisfaction and increased app success.
+
+Here are some of the benefits of using crash reporting services:
+
+- Identify and fix bugs quickly and efficiently.
+- Prioritize their efforts and focus on the most critical problems.
+- Improve the user experience by reducing the number of crashes and errors.
+- Collect feedback from users about their experiences with the app.
+
+## What does it do
+
+Crashlytics is a crash reporting and analysis tool provided by Firebase. Crashlytics helps developers track and understand crashes that occur in their mobile applications. It provides insights into the causes of crashes and allows developers to take action to improve the stability of their apps. Here's what Crashlytics does:
+
+**1. Crash Reporting:** Crashlytics monitors your mobile app in real-time and captures crash reports when the app encounters an unexpected error or crashes. When a crash occurs, Crashlytics collects information about the crash, including the stack trace, device information, app version, and more.
+
+**2. Error Insights:** Crashlytics provides detailed crash reports that help you understand the circumstances under which the crash occurred. This includes information about the sequence of events leading up to the crash, the specific lines of code that were executing, and any variables involved.
+
+**3. Real time Alerts:** Crashlytics can send you real-time alerts when new crashes are detected in your app. This allows you to be aware of issues as they arise and respond quickly to minimize the impact on users.
+
+**4. Custom Logs:** In addition to crash reports, Crashlytics allows you to log custom events and data. This can be helpful for tracking user actions, debugging, and gaining insights into the usage patterns of your app.
```

Comment:
WE should first cover how to send the crash logs and then move this to next heading below. As sending custom logs

Reply:
updated